common/tools/dev-tool/src/commands/samples/publish.ts

```diff
@@ -383,6 +385,19 @@ async function makeSampleGenerationInfo(
               }
 
               current[dependency] = dependencyVersion;
+              // It would be really weird to depend on `@types/*` in a source file but if we did
+              // it'd be handled above.
+              if (dependency.indexOf("@types/") !== 0) {
```

Comment:
NIT: 
```suggestion
              if (dependency.indexOf("@types/") === -1) {
```

Reply:
For what it's worth, I'm specifically trying to make sure that the dependency name doesn't _start_ with `@types`, which is why I do the `!== 0` check.